### ISSUE TITLE
Karma exits with code 0 when the coverage threshold is below the configuration

### ISSUE THREAD

**Issue by jesusbarcenilla** [karma-runner/karma-coverage#421]
Hi,

I added this configuration to my karma.conf.js 
```coverageReporter: {
            type: 'html',
            dir: 'test/coverage/',
            check: {
                global: {
                    statements: 85,
                    branches: 70,
                    functions: 85,
                    lines: 85
                }
            }
```

The logs work fine and if the thresholds are not met, I can see error logs in the terminal but karma always exit wit code 0. That affect my CI process that is not able to detect something is wrong.

The documentation says 
> If the thresholds are not met, karma will return failure.

and I've review the code and I've seen the plugin is setting the exit code to 1 so I'm not sure if it is a issue in karma-coverage or in karma itself. 

I'm using : 
karma@5.1.1
karma-coverage@2.0.3

**Comment by Epoxide** [comment]
I'm experiencing the same.

`karma.conf.js`

```js
reporters: ['coverage', 'spec'],

 coverageReporter: {
    includeAllSources: true,
    dir: 'coverage/',
    reporters: [
         { type: "html", subdir: "html" },
         { type: 'text' },
         { type: 'text-summary' }
     ],
     check: {
          global: {
              statements: 80,
              lines: 80,
              branches: 80,
              functions: 80
          }
     }
 },
```

Karma output:

```bash
07 08 2020 13:41:20.282:ERROR [coverage]: Chrome Headless 85.0.4182.0 (Linux x86_64): Coverage for branches (66.96%) does not meet global threshold (80%)
=============================== Coverage summary ===============================
Statements   : 81.5% ( 1674/2054 )
Branches     : 66.96% ( 612/914 )
Functions    : 85.93% ( 281/327 )
Lines        : 81.63% ( 1653/2025 )
```

Karma exits cleanly with 0. I've tried `emitWarning: false` inside `check` with no luck (even though this is not documented in `karma-coverage`). I'm running:

```
karma: 5.1.1
karma-coverage: 2.0.3
```

**Comment by matz3** [comment]
See https://github.com/karma-runner/karma-coverage/issues/418
Issue is still not solved

**Comment by jesusbarcenilla** [comment]
@matz3 It looks like your PR was already merged in master. I've tested it and it works fine. 
Looking forward for the next karma release.
I'll keep this issue open until then.

Thank you very much.

**Comment by matz3** [comment]
Solved with [karma v5.2.0](https://github.com/karma-runner/karma/releases/tag/v5.2.0) (in combination with karma-coverage v2.0.3)

**Comment by anthony-redFox** [comment]
Thanks @matz3
